semgrep_output_v1.atd

```diff
@@ -956,4 +956,5 @@ type finding <ocaml attr="deriving show"> = {
   ?fixed_lines: string list option;
   ?sca_info: sca_info option;
   ?dataflow_trace: cli_match_dataflow_trace option;
+  ?product: string option;
```

Comment:
it's useful to also add in comment since which version of semgrep the field was introduced.